python/core/unified_api_server.py

```
@@ -7996,6 +8316,297 @@ def _get_mock_analysis_results() -> List[GameAnalysisSummary]:
+
+    try:
+        result = await stripe_service.create_checkout_session(
+            user_id=user_id,
+            tier_id=tier_id,
+            credit_amount=credit_amount,
+            success_url=success_url,
+            cancel_url=cancel_url
+        )
+
+        if 'error' in result:
+            raise HTTPException(status_code=400, detail=result['error'])
```

Ruff (0.14.2)

Abstract `raise` to an inner function

(TRY301)